document/CoreFormat.mdk

```diff
@@ -303,51 +316,116 @@ an analysis run. The value of `invocationInfo` would have to be combined with in
 ~
 
-The `analysisTargets` array shall be empty if no analysis targets were scanned in the course of the run.
+Each property name in the `fileInfo` object shall be the URI of a scanned file.
+No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
 
-~ IsoNumberedNote
-This could happen if the command line specified a wildcard such as *.cc for the input files,
-and no files matched the wildcard.
-~
+Each property value in the `fileInfo` object shall be an array of `fileReference` objects (&sect;[#fileReference])
+which contain information about the file identified by the URI in the property name.
 
-~ IsoExample
+If the scanned file is not nested within another file, then the array shall consist of a single
+element, and the property name shall be the absolute URI of the scanned file.
+
+~ IsoNumberedExample { @IsoNumberedExample=1}
 
 ```
-[
-    {
-        "uri": "file:///C:/Code/a.cc",
-        "hashes": [
+runInfo: {
+    fileInfo: {
+        "file:///C:/Code/main.cc": [
             {
-                "value": "b13ce2678a8807ba0765ab94a0ecd394f869bc81",
-                "algorithm": "SHA-256"
+                "uri": "file:///C:/Code/main.cc"
+                "mimeType": "text/x-c",
+                "hashes": [
+                    {
+                        "value": "b13ce2678a8807ba0765ab94a0ecd394f869bc81",
+                        "algorithm": "SHA-256"
+                    }
+                ],
+                "tags": [ "entryPoint" ]
             }
         ]
-    },
+    }
+}
+```
+~
 
-    {
-        "uri": "file:///C:/Code/b.cc",
-        "hashes": [
+If the scanned file is nested within another file (for example, within a compressed container),
+then the first `fileReference` object in the array shall refer to the outermost container,
+the next one shall refer to a file nested immediately within that container, and so on,
+with the last element of the array referring to the most deeply nested file.
+
+In this case, the property name shall be the absolute URI of the outermost containing
+file, together with a fragment that describes the nesting of the scanned file within
+its parent or parents.
+If the scanned file is nested more than one level deep in the outermost containing file,
+the fragments representing each level of nesting may be combined in any way desired,
+as long as no two of the resulting property names are equivalent as defined in &sect;[#uriProperties].
+
+~ IsoNumberedExample
+Consider a Flash object contained in a word processing document which is in turned contained
+in a Zip archive.
+Suppose the path to the word processing document within the Zip archive is `/docs/intro.doc`,
+and suppose the Flash object is located at offset `0x17f2` from the start of the word processing
+document.
+Then one possible value for the property name within the `fileInfo` object would be:
+
+```
+file:///C:/Code/presentation.zip#/docs/intro.doc-0x17f2
```

Comment:
This offset is not the region. It is the offset of the scanned file (in this case, the Flash object), within its parent. The idea in this example is that an issue has been found _within_ the Flash object. The `region` property of the `result.locations[0].resultFile` object would tell the location within the Flash object where the issue was detected. If the issue were merely the _presence_ of the Flash object, you'd be right. I will clarify the example.